name-badges/makebadges.py

```diff
@@ -0,0 +1,35 @@
+#!/bin/pyhton3
+
+import sys
+import re
+import lxml.etree as ET
+import subprocess
+#from cairosvg import svg2png
+
+namesfile = sys.argv[1]
+
+names=open(namesfile).readlines()
+
+def altersvg(file,name,role):
+    svg = open(file, mode="rb").read()
+    root = ET.fromstring(svg)
+    print(root)
+    root.findall(".//text[@inkscape:label='Name']",root.nsmap)[0][0].text=name
+    root.findall(".//text[@inkscape:label='Role']",root.nsmap)[0][0].text=role
+    return(ET.tostring(root, encoding='UTF-8'))
+
+
+
+subprocess.call(["mkdir", "tmp"])
+subprocess.call(["mkdir", "out"])
+for name in names:
+    parts = name.strip().split(":")
+    print(parts)
+    customsvg = altersvg("badge.svg",parts[0],parts[1])
```

Comment:
We should strip trailing spaces from the parts here to ensure spaces don't really matter in the input file